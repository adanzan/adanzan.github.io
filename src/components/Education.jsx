/* eslint no-unused-vars: 0 */

import * as React from "react";

import styles from "../styles/Education.module.css";

import EducationSnippet from "./EducationSnippet";

import middleburyLogo from "../resources/images/middleburyLogo.png";
import orchlonLogo from "../resources/images/orchlonLogo.png";

export default function Education() {
	const middleburyAchievements = ["cum laude", "Computer Science Honors"];
	const orchlonAchievements = ["Captain of varsity volleyball team", "Class President for 9 years"];

	return (
		<div id="education" className={styles.Education}>
			<h2>EDUCATION</h2>
			<EducationSnippet
				name="Middlebury College"
				degree={`B.A. in Computer Science,
                                         \n     Minor in Russian Language`}
				dateRange="Sep 2018 - Feb 2023"
				location="Middlebury, VT"
				logo={middleburyLogo}
				link="https://www.middlebury.edu/"
				gpa="3.5"
				achievements={middleburyAchievements}
			/>
			<EducationSnippet
				name="Orchlon International School"
				degree="High School Diploma"
				dateRange="Sep 2006 - May 2018"
				location="Ulaanbaatar, Mongolia"
				logo={orchlonLogo}
				link="https://orchlon.mn/en/"
				gpa="3.9"
				achievements={orchlonAchievements}
			/>
		</div>
	);
}
