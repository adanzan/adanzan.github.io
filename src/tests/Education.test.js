/* eslint no-unused-vars: 0 */
import * as React from "react";
import { render } from "@testing-library/react";

import Education from "../components/Education"; 

let testRender;

describe(("Education component"), () => {
	beforeEach(() => {
		testRender = render(<Education />);
	});
	test("matches snapshot", () => {
		expect(testRender).toMatchSnapshot();
	});
	test("has title", () => {
		expect(testRender.getByText("EDUCATION"));
	});
});
