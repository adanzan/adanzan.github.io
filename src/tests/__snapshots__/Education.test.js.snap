// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education component matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Education"
        id="education"
      >
        <h2>
          EDUCATION
        </h2>
        <div
          class="EducationSnippet"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation10 snippetPaper css-14epxft-MuiPaper-root"
          >
            <div
              class="org"
            >
              <a
                href="https://www.middlebury.edu/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt="Middlebury Collegelogo"
                  class="logo"
                  src="middleburyLogo.png"
                />
              </a>
              <div
                class="nameLocationWrapper"
              >
                <div
                  class="nameLocation"
                >
                  <a
                    class="name"
                    href="https://www.middlebury.edu/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <h3>
                      Middlebury College
                    </h3>
                  </a>
                  <p
                    class="location"
                  >
                    Middlebury, VT
                  </p>
                </div>
              </div>
            </div>
            <div
              class="detailsWrapper"
            >
              <div
                class="details"
              >
                <p
                  class="detailsDegree"
                >
                   
                  <strong>
                    B.A. in Computer Science,
                                         
     Minor in Russian Language
                  </strong>
                   
                </p>
                <p
                  class="detailsDate"
                >
                  Sep 2018 - Feb 2023
                </p>
                <p>
                   
                  <strong>
                    GPA:
                  </strong>
                   
                  3.5
                </p>
                <p>
                  <li>
                    cum laude
                  </li>
                  <li>
                    Computer Science Honors
                  </li>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="EducationSnippet"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation10 snippetPaper css-14epxft-MuiPaper-root"
          >
            <div
              class="org"
            >
              <a
                href="https://orchlon.mn/en/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt="Orchlon International Schoollogo"
                  class="logo"
                  src="orchlon.png"
                />
              </a>
              <div
                class="nameLocationWrapper"
              >
                <div
                  class="nameLocation"
                >
                  <a
                    class="name"
                    href="https://orchlon.mn/en/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    <h3>
                      Orchlon International School
                    </h3>
                  </a>
                  <p
                    class="location"
                  >
                    Ulaanbaatar, Mongolia
                  </p>
                </div>
              </div>
            </div>
            <div
              class="detailsWrapper"
            >
              <div
                class="details"
              >
                <p
                  class="detailsDegree"
                >
                   
                  <strong>
                    High School Diploma
                  </strong>
                   
                </p>
                <p
                  class="detailsDate"
                >
                  Sep 2006 - May 2018
                </p>
                <p>
                   
                  <strong>
                    GPA:
                  </strong>
                   
                  3.9
                </p>
                <p>
                  <li>
                    Captain of varsity volleyball team
                  </li>
                  <li>
                    Class President for 9 years
                  </li>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Education"
      id="education"
    >
      <h2>
        EDUCATION
      </h2>
      <div
        class="EducationSnippet"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation10 snippetPaper css-14epxft-MuiPaper-root"
        >
          <div
            class="org"
          >
            <a
              href="https://www.middlebury.edu/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Middlebury Collegelogo"
                class="logo"
                src="middleburyLogo.png"
              />
            </a>
            <div
              class="nameLocationWrapper"
            >
              <div
                class="nameLocation"
              >
                <a
                  class="name"
                  href="https://www.middlebury.edu/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <h3>
                    Middlebury College
                  </h3>
                </a>
                <p
                  class="location"
                >
                  Middlebury, VT
                </p>
              </div>
            </div>
          </div>
          <div
            class="detailsWrapper"
          >
            <div
              class="details"
            >
              <p
                class="detailsDegree"
              >
                 
                <strong>
                  B.A. in Computer Science,
                                         
     Minor in Russian Language
                </strong>
                 
              </p>
              <p
                class="detailsDate"
              >
                Sep 2018 - Feb 2023
              </p>
              <p>
                 
                <strong>
                  GPA:
                </strong>
                 
                3.5
              </p>
              <p>
                <li>
                  cum laude
                </li>
                <li>
                  Computer Science Honors
                </li>
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="EducationSnippet"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation10 snippetPaper css-14epxft-MuiPaper-root"
        >
          <div
            class="org"
          >
            <a
              href="https://orchlon.mn/en/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Orchlon International Schoollogo"
                class="logo"
                src="orchlon.png"
              />
            </a>
            <div
              class="nameLocationWrapper"
            >
              <div
                class="nameLocation"
              >
                <a
                  class="name"
                  href="https://orchlon.mn/en/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <h3>
                    Orchlon International School
                  </h3>
                </a>
                <p
                  class="location"
                >
                  Ulaanbaatar, Mongolia
                </p>
              </div>
            </div>
          </div>
          <div
            class="detailsWrapper"
          >
            <div
              class="details"
            >
              <p
                class="detailsDegree"
              >
                 
                <strong>
                  High School Diploma
                </strong>
                 
              </p>
              <p
                class="detailsDate"
              >
                Sep 2006 - May 2018
              </p>
              <p>
                 
                <strong>
                  GPA:
                </strong>
                 
                3.9
              </p>
              <p>
                <li>
                  Captain of varsity volleyball team
                </li>
                <li>
                  Class President for 9 years
                </li>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
